configurations/elman.json

```
@@ -0,0 +1,42 @@
+{
+  "environment": "LunarLander-v2",
+  "random_seed": 0,
+  "number_generations": 200,
+  "optimizer": {
+    "type": "CMA_ES",
+    "population_size": 100,
+    "sigma": 1.0,
+    "checkpoint_frequency": 0,
+    "hof_size": 5,
+    "mu": 25,
+    "efficiency_weight": 0.0,
+    "fix_seed_for_generation": false
```

```suggestion
    "fix_seed_for_generation": true
```

Ist empfohlen laut Log des Programms:

```
WARNING: When using novelty you should also set fix_seed_for_generation to true. 
```

---

erledigt